package.json

```diff
@@ -9,7 +9,7 @@
     "good": "^7.0.2",
     "good-console": "^6.3.1",
     "good-squeeze": "^5.0.0",
-    "hapi": "^16.1.0",
```

Comment:
Please add a reason for updating the hapi version? Are you using hapi features that are not available earlier?

Also, this should be a separate commit.